wagtail/admin/widgets/button.py

```diff
@@ -48,12 +48,18 @@ def __eq__(self, other):
                 and self.priority == other.priority)
 
 
-class PageListingButton(Button):
+# Base class for all listing buttons
+# This is also used by SnippetListingButton defined in wagtail.snippets.widgets
+class ListingButton(Button):
     def __init__(self, label, url, classes=set(), **kwargs):
         classes = {'button', 'button-small', 'button-secondary'} | set(classes)
         super().__init__(label, url, classes=classes, **kwargs)
 
 
+class PageListingButton(ListingButton):
```

Comment:
Yep, this is to keep existing imports happy. I also added a `SnippetListingButton` variant too for consistency (in case they have to diverge in the future)